go.mod

```diff
@@ -1,14 +1,13 @@
 module github.com/percona/everest
 
-go 1.23.2
+go 1.23.4
 
 replace (
 	github.com/containerd/containerd v1.7.12 => github.com/containerd/containerd v1.7.13
-	k8s.io/api => k8s.io/api v0.32.0
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.32.0
-	k8s.io/apimachinery => k8s.io/apimachinery v0.32.0
-	k8s.io/client-go => k8s.io/client-go v0.32.0
-	k8s.io/component-base => k8s.io/component-base v0.32.0
+	k8s.io/api => k8s.io/api v0.31.1
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.31.1
+	k8s.io/client-go => k8s.io/client-go v0.31.1
+	k8s.io/component-base => k8s.io/component-base v0.31.1
```

Comment:
Probably you didn't mean to downgrade these

Reply:
Right, thanks for pointing that out, merge didn't went correct.